testutil/testnode/full_node.go

```diff
@@ -214,12 +215,34 @@ func DefaultNetwork(t *testing.T, blockTime time.Duration) (accounts []string, c
 		t.Log("tearing down testnode")
 		require.NoError(t, stopNode())
 		require.NoError(t, cleanupGRPC())
-		require.NoError(t, os.RemoveAll(tmCfg.RootDir))
+		require.NoError(t, removeDir(path.Join([]string{tmCfg.RootDir, "config"}...)))
 	})
 
 	return accounts, cctx
 }
 
+// removeDir removes the directory `rootDir`.
+// The main reason for using it is to know if some file is used by some leaking process during
+// cleanup and be able to identify where the leak is occurring.
+// TODO: remove after fixing the CI flakiness
```

Comment:
>TODO: remove after fixing the CI flakiness

does this fix some of the flakiness or is this just a hacky fix?

Reply:
Its a way to know exactly which file is causing the flakiness to investigate which process is leaking. Next time the flakiness happens, this should give us more informatjon (might even fix the flakiness, but not sure)